api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -8,7 +8,7 @@
 
 
 
-
+    <bean id="bedTagValidator" class="org.openmrs.validator.BedTagValidator"/>
```

Comment:
Is there a reason why you are duplicating this bean by declaring it in xml and also annotating the class with `Component`?